jsk_perception/scripts/install_trained_data.py

```diff
@@ -222,6 +222,21 @@ def main():
         md5='47a507934b6bc20f0d9274825b734942',
     )
 
+    # node_scripts/mask_rcnn_instance_segmentation.py 73B2 kitchen
```

Comment:
@taichiH 
please add link to classneames.yaml to here, also create page to show trained data within 'https://drive.google.com/uc?id=1f9E2b1uJrk3eLbJmEDSSPl6TQKgKvZLW directory

Reply:
@k-okada  sorry, I don't understand meaning about "create page". Where should I create a page? Do you say add model to google drive and show recognition result on this pr thread?